Python/PyPlasma.h

```diff
@@ -356,10 +360,12 @@ template <> inline unsigned char pyPlasma_get(PyObject* value) { return (unsigne
 template <> inline uint16_t pyPlasma_get(PyObject* value) { return (uint16_t)(unsigned long)PyInt_AsLong(value); }
 template <> inline uint32_t pyPlasma_get(PyObject* value) { return (uint32_t)(unsigned long)PyInt_AsLong(value); }
 template <> inline uint64_t pyPlasma_get(PyObject* value) { return (uint64_t)(unsigned long)PyInt_AsLong(value); }
+template <> inline unsigned long pyPlasma_get(PyObject* value) { return (unsigned long)PyInt_AsLong(value); }
 template <> inline signed char pyPlasma_get(PyObject* value) { return (signed char)PyInt_AsLong(value); }
 template <> inline int16_t pyPlasma_get(PyObject* value) { return (int16_t)PyInt_AsLong(value); }
 template <> inline int32_t pyPlasma_get(PyObject* value) { return (int32_t)PyInt_AsLong(value); }
 template <> inline int64_t pyPlasma_get(PyObject* value) { return (int64_t)PyInt_AsLong(value); }
+template <> inline long pyPlasma_get(PyObject* value) { return (long)PyInt_AsLong(value); }
```

Comment:
You can't really mix `int**_t` and `long` safely in overloads, as you've no doubt discovered...  Probably the safest fix (and the one I ended up going with for `string_theory`) is to use the built-in types exclusively.  That is, update all the overloads from `[u]int##_t` to `[unsigned] short/int/long/long long`.  You'll also want to be sure to update the casts where appropriate.

Reply:
Hopefully that won't cause issues elsewhere in the bindings. Okay, I'll try that soon.